tests/e2e/walletFeatures.spec.ts

```diff
@@ -94,4 +88,31 @@ test.describe("Wallet features", () => {
 
     await browser.close();
   });
+
+  test("auth via LNURL", async () => {
+    const { page, browser, extensionId } = await loadExtension();
+    const $optionsdocument = await unlockExtension({
+      page,
+      extensionId,
+    });
+
+    // go to send
+    await (await findByText($optionsdocument, "Send")).click();
+    const invoiceInput = await findByText(
+      $optionsdocument,
+      "Invoice, Lightning Address or LNURL"
+    );
+    await invoiceInput.type(
+      "lightning:LNURL1DP68GURN8GHJ7MRWW4EXCTNXD9SHG6NPVCHXXMMD9AKXUATJDSKKCMM8D9HR7ARPVU7KCMM8D9HZV6E384NXVVEKXSEXYWRYX3SN2EPCX5EKYV35VVEK2VNPXESKZVN9XVCX2VNYX5CXYCFHXUCRJV3JX5UNJV3J8Q6N2WTYVS6NGVE48Q6X2VP3YCHUNC"
```

Comment:
this is problematic because it is doing an HTTP request to an external source.
maybe just put the some JSON in a gist that we control and create an encoded lnurl for that url then.